frontend/nginx.conf

```
@@ -16,11 +16,22 @@ http {
 
     access_log /var/log/nginx/nginx-frontend.log;
 
+    gzip on;
+    gzip_types text/plain text/css application/json application/javascript text/xml application/xml application/xml+rss text/javascript;
```

```suggestion
    gzip_types text/plain text/css application/json text/xml application/xml application/xml+rss text/javascript;
```

`application/javascript` is considered legacy, it's probably safe to remove it. 
`application/xml+rss`: We don't have RSS content